### PR TITLE
AT_01.18.01 | Verify the link "Register" is clickable and redirects t…

### PR DESCRIPTION
**Pull request by qa-tester22**
…o the "Register agent account" popup'
https://trello.com/c/2rSh7HIh/226-at011801-start-page-restore-password-popup-footer-ui-and-functionality-verify-the-link-register-is-clickable-and-redirects-to-th

### CODE REVIEW

File: cypress/e2e/testUiAndFunction/startPage/US_01.18_ RestorePasswordPopupFooterUIAndFunc.cy.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/// <reference types="Cypress"/>
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/startPage/US_01.18_ RestorePasswordPopupFooterUIAndFunc.cy.js:1]: The file name must not contain spaces.

File: cypress/fixtures/startPage.json [hunk 2]
```diff
@@ -44,5 +44,9 @@
     },
     "links": {
         "registerAccountNowEnglishText": "Register account now"
+    },
+
+    "header": {
```
Comment (Rom-T) [inline cypress/fixtures/startPage.json:49]: We already have a headers on line 20

File: cypress/pageObjects/StartPage.js [hunk 3]
```diff
@@ -71,7 +71,8 @@ export class RestorePopup {
     getMessageAlert = () => cy.get('#restoreModal div.alert');
     getRestorePasswordHeader = () => cy.get('#restoreModal h2');
     getCloseButton = () => cy.get('div#restoreModal .modal-header .close');
-    getRestorePopup = () => cy.get('div#restoreModal')
+    getRestorePopup = () => cy.get('div#restoreModal');
+    getRestorePopupRegisterLink = () => cy.get('.form-horizontal .modal-footer .pull-left a');
```
Comment (Rom-T) [inline cypress/pageObjects/StartPage.js:75]: Please rename to getRegisterLink

File: cypress/pageObjects/StartPage.js [hunk 4]
```diff
@@ -85,6 +86,10 @@ export class RestorePopup {
 
     clickCloseButton(){
         this.getCloseButton().click()
+    };
+
+    clickRestorePopupRegisterLink(){
```
Comment (Rom-T) [inline cypress/pageObjects/StartPage.js:91]: Please rename to clickRegisterLink()

Comment (qa-tester22) [reply cypress/pageObjects/StartPage.js]: I've done it.

**Review by MariaMoroz — APPROVED** [review]
Лучше заранее вынести логин с визитом в before вместо beforeEach

**Pull request merged**